src/server/pfs/pfs.go

```diff
@@ -303,6 +328,7 @@ var (
 	commitOnOutputBranchRe    = regexp.MustCompile("cannot start a commit on an output branch")
 	squashWithoutChildrenRe   = regexp.MustCompile("cannot squash a commit that has no children")
 	dropWithChildrenRe        = regexp.MustCompile("cannot drop a commit that has children")
+	deleteWithDependentSetsRe = regexp.MustCompile("cannot be squashed in isolation. To delete them, also squash")
```

Comment:
This whole regexp-matching of error strings business seems horribly brittle and like something we should move away from rather than perpetuate.